src/components/BookForm.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { addBook } from '../redux/books/books';
+
+const BookForm = () => {
+  const [title, setTitle] = useState('');
+  const dispatch = useDispatch();
+  const handleChange = (event) => {
+    setTitle(event.target.value);
+  };
+  const handleAddBook = () => {
+    const newBook = {
+      name: title,
+      author: 'Anonymous',
```

Comment:
- [ ] In this step `author` should not be set to a default of "anonymous". Just like with the `title`, we should use `useState` and have a field where we can type the author's name in the form. Please refactor this.

  See what the requirements say about this 👇🏿:

  > - In your React component responsible for adding new books **_`set the data inputs in the local React state (set title and author`_** - NOTE: categories and comments are NOT part of this step).